misc/acrn-config/board_config/README

```diff
@@ -1,5 +1,6 @@
 Please run board_cfg_gen.py to generate board related configuration patch, the patch would be applied on current acrn-hypervisor git tree automatically.
 
-usage: python3 board_cfg_gen.py [h] --board <board_info_file>
+usage: python3 board_cfg_gen.py [h] --board <board_info_file> --scenario <scenario info file>
```

Comment:
scenario info file -> scenario_info_file to align with the reset part

Reply:
agree, will fix it in next version